includes/in-person-payments/class-wc-payments-in-person-payments-receipt-service.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Class WC_Payments_In_Person_Payment_Print_Receipt_Service
+ *
+ * @package WooCommerce\Payments
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class handling in person payments receipts.
+ */
+class WC_Payments_In_Person_Payments_Receipt_Service {
+
+	/**
+	 * Renders the receipt template.
+	 *
+	 * @param  array    $settings Merchant settings.
+	 * @param  WC_Order $order Order instance.
+	 * @param  array    $charge Charge data.
+	 *
+	 * @return void
+	 */
+	public static function render_receipt( array $settings, WC_Order $order, array $charge ) {
```

Comment:
Let's turn it into a proper service method by capturing the content (`ob_start`, `ob_get_clean`) and returning it as string data. The controller can render it then, as part of the controller's responsibilities.

Reply:
Done!